msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -209,6 +209,16 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+### Office for Android
```

Comment:
Suggest to conform to the branding guidelines for Office, Outlook, and so on. This is a global comment.